ProjectSettings/ProjectSettings.asset

```diff
@@ -382,8 +382,9 @@ PlayerSettings:
   enableInternalProfiler: 0
   logObjCUncaughtExceptions: 1
   enableCrashReportAPI: 0
-  cameraUsageDescription: AR BABY
-  locationUsageDescription: 
+  cameraUsageDescription: Place Invaders needs camera access to display AR content.
+  locationUsageDescription: Place Invaders needs location access to display multiplayer
```

Comment:
Multi-player rooms close to you?

Reply:
Right, that makes sense. I will update that then merge up.